deployment/agent/flyte_agent_helm_generated.yaml

```diff
@@ -78,7 +78,7 @@ spec:
         - pyflyte
         - serve
         - agent
-        image: "ghcr.io/flyteorg/flyteagent:1.10.7"
+        image: "ghcr.io/flyteorg/flyteagent:1.10.8b2"
```

Comment:
```suggestion
        image: "ghcr.io/flyteorg/flyteagent:1.10.8b1"
```
right now, b1 is available and b2 is not.
<img width="1270" alt="image" src="https://github.com/flyteorg/flyte/assets/76461262/acde17fe-579c-45ae-8ca2-f40e6096db55">

https://github.com/flyteorg/flytekit/pkgs/container/flyteagent

Reply:
or we should release a new flyteagent image.

Reply:
I am going to sleep later, if this PR works well, it will be great to change the image name.
Else, maybe we didn't use the probe api correctly.
https://github.com/flyteorg/flyte/pull/5014

Reply:
I am not sure why I need to pull the flyteagent image for such long time.
Maybe because I live in a place with bad internet connection.
So create a PR above to help test this.
<img width="1252" alt="image" src="https://github.com/flyteorg/flyte/assets/76461262/ee156d4a-6edf-4f91-ba09-cb8d4991322f">

Reply:
Update:
It works, but need to wait 5 minutes
<img width="1256" alt="image" src="https://github.com/flyteorg/flyte/assets/76461262/c2f229ea-74b7-4dc7-8439-285cfa1fb314">

Reply:
No really sure why gRPC probe doesn't turn flyteagent ready.
Maybe it is because of the health check service.

Reply:
<img width="1280" alt="image" src="https://github.com/flyteorg/flyte/assets/76461262/c1008dc2-bf69-443c-b63f-3b27cdc81b5c">

Reply:
https://github.com/flyteorg/flytekit/actions/runs/8168720389 failed to push image for some reason

Reply:
they can remove the grpc probe from the chart if they use the old image.